src/pudl/dfc.py

```diff
@@ -94,7 +94,7 @@ def store(self, name: str, data: pd.DataFrame):
         if name in self._table_ids:
             raise TableExists(f'Table {name} already present in the DFC.')
         filename = self._get_filename(name, self._instance_id)
-        if not filename.startswith("gs://"):
+        if not (filename.startswith("gs://") or filename.startswith("s3://")):
```

Comment:
This will be rendered obsolete by the changes in here: https://github.com/catalyst-cooperative/pudl/pull/916